otherlibs/stdlib_alpha/effect.ml

```diff
@@ -392,8 +392,14 @@ type ('a, 'e) perform = ('a, 'e) op * 'e Handler.t'
 
 external perform : ('a, 'e) perform -> 'a = "%perform"
 
+(* Stacks are represented as tagged pointers, and garbage collection
+   invalidates them. Therefore, no allocations must happen between
+   creation and use of a [stack]. *)
 type (-'a, +'b) stack : immediate
 
+(* Stacks are represented as tagged pointers, and garbage collection
```

Comment:
...and `last_fiber`s are not relied upon to keep fibers alive, since there should always be another reference via `cont.fiber(->parent)*`